prov/sockets/src/sock_ep.c

```diff
@@ -582,7 +582,7 @@ static ssize_t sock_tx_size_left(struct fid_ep *ep)
 		return -FI_EOPBADSTATE;
 
 	fastlock_acquire(&tx_ctx->wlock);
-	num_left = rbavail(&tx_ctx->rb)/SOCK_EP_TX_ENTRY_SZ;
+	num_left = ofi_rounddown_power_of_two(rbavail(&tx_ctx->rb)/SOCK_EP_TX_ENTRY_SZ);
```

Comment:
I'm not sure I understand this. So the return value of `fi_tx_size_left` will always be a power of two after this change?